controllers/articles-controller.js

```diff
@@ -61,3 +62,19 @@ exports.postComment = (req, res, next) => {
       next(err);
     });
 };
+
+exports.updateArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const newVote = req.body;
+  return Promise.all([
+    updateArticleVotes(article_id, newVote),
+    selectArticle(article_id),
```

Comment:
is this to check the article exists? since we're not using the value of the resolved promise a checkExists function might be better, but this is fine

Reply:
yes! it also sorts out the error messages for me if the article doesn't exist, since this is already defined inside selectArticle.... not sure if the correct way though! i have also realised that i could just use psql errors for this (i think!!) but i'm in too deep now!!